clang/test/SemaSYCL/prohibit-thread-local.cpp

```diff
@@ -0,0 +1,49 @@
+// RUN: %clang_cc1 -fsycl-is-device -verify -fsyntax-only -std=c++17 %s
+
+thread_local const int prohobit_ns_scope = 0;
+thread_local int prohobit_ns_scope2 = 0;
+thread_local const int allow_ns_scope = 0;
+
+struct S {
+  static const thread_local int prohibit_static_member;
+  static thread_local int prohibit_static_member2;
+};
+
+struct T {
+  static const thread_local int allow_static_member;
```

Comment:
This one too.

Reply:
Attempt to show that we're not intending to diagnose there.